docs/siem/siem-ui.asciidoc

```diff
@@ -50,6 +50,22 @@ or the Detections API.
 [role="screenshot"]
 image::siem/images/detections-ui.png[]
 
+[float]
+[[cases-ui]]
+=== Cases (Beta)
```

Comment:
suggest removing Beta from the title and using this in the first line:

beta:[] Cases are used to open...

That way, you will get a definition of what it means for a feature to be in beta.  If you do that, you'll also have to make the change for Detections earlier in the page.

If you keep beta in the title, we use beta (lower case b)